src/collector/parsers/typescript.ts

```diff
@@ -239,13 +240,26 @@ function createAccessors(node: ts.AccessorDeclaration): IAccessor {
     }
 }
 
-function createProperty(node: ts.PropertyDeclaration): IProperty {
-    return {
-        name: node.name ? node.name.getText() : '',
-        description: extractDescription(node),
-        returnType: extractReturnValue(node),
-        visibility: extractVisibility(node),
-        isReadonly: isReadonly(node)
+function createProperty(isInternal = false) {
+    return function (node: ts.PropertyDeclaration): IProperty {
+        const isPublicMethod = VisibilityMap[ts.SyntaxKind.PublicKeyword] === extractVisibility(node);
+        const propertyOutput = {
+            name: node.name ? node.name.getText() : '',
+            description: extractDescription(node),
+            returnType: extractReturnValue(node),
+            visibility: extractVisibility(node),
+            isReadonly: isReadonly(node)
+        };
+
+        if (!isInternal && isPublicMethod) {
+            return propertyOutput;
+        }
+
+        if (isInternal && !isPublicMethod) {
+            return propertyOutput;
+        }
```

Comment:
Maybe it will be better
if ((!isInternal && isPublicMethod) || (isInternal && !isPublicMethod)) {
    return propertyOutput;
}
instead of two if blocks